deploy/cluster_role.yaml

```diff
@@ -181,21 +192,22 @@ rules:
   resources:
   - deployments
   verbs:
-  - 'get'
-  - 'list'
-  - 'create'
```

Comment:
Did you mean to drop `create` here?

Reply:
Yes, these are for the `inject` feature, so, we just want to change existing deployments, not create new ones. But you made me realize that this doesn't make sense, as we have full right for `deployments` defined a few lines above... 

Created #1130 to track this.